js/main.js

```diff
@@ -0,0 +1,36 @@
+const div = document.createElement('div');
+const grayBgDiv = document.createElement('div');
+const hamburgerBttn = document.querySelector('.menu-bars');
+
+hamburgerBttn.addEventListener('click', () => {
+  div.id = 'overlay';
+  div.classList.add('overlay');
+  grayBgDiv.id = 'overlay-menu-bg';
+  grayBgDiv.classList.add('overlay');
+  div.innerHTML = '<img src="./media/cancel.png" class="cancelBttn"></img>'
+    + '<ul class="mobile-menu-ul">'
+    + '<li class="mobile-menu-li"><a href="#projects">Porfolio</a></li>'
+    + '<li class="mobile-menu-li"><a href="#about-me">About</a></li>'
+    + '<li class="mobile-menu-li"><a href="#contact">Contact</a></li>'
+    + '</ul>';
```

Comment:
- [ ] Please fix the link in the menu for the contact me section.
The href it's not working because the 'id' for the contact section in the index.html: `contact-section` doesn't match with the `#contact` inside the main.js

Reply:
Done ✔

Reply:
Well addressed 👌👌